spring-cloud-open-service-broker-autoconfigure/src/main/java/org/springframework/cloud/servicebroker/autoconfigure/web/Metadata.java

```diff
@@ -0,0 +1,82 @@
+package org.springframework.cloud.servicebroker.autoconfigure.web;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Metadata for service plans.
+ * Holds keys specified in cloudfoundry profile in OSB specs,
+ * as well as custom non-standard keys.
+ */
+public class Metadata {
+
+    private List<Cost> costs = new ArrayList<Cost>();
+
+    private List<String> bullets = new ArrayList<>();
+
+    private String displayName;
+
+    /**
```

Comment:
It might be possible to use the `@JsonAnySetter` as is done in [BindResource](https://github.com/spring-cloud/spring-cloud-open-service-broker/blob/master/spring-cloud-open-service-broker-core/src/main/java/org/springframework/cloud/servicebroker/model/binding/BindResource.java#L41-L46) to avoid the `extra` key in the hash.